ssl/ssl_locl.h

```diff
@@ -11,12 +11,12 @@
 
 #ifndef HEADER_SSL_LOCL_H
 # define HEADER_SSL_LOCL_H
+# include "e_os.h"              /* struct timeval for Windows */
 # include <stdlib.h>
 # include <time.h>
 # include <string.h>
 # include <errno.h>
 
-# include "e_os.h"
 # if defined(__unix) || defined(__unix__)
 #  include <sys/time.h>         /* struct timeval for DTLS */
 # endif
```

Comment:
On side note (and this is more like a mental note for myself) one can wonder if it's appropriate to move this to e_os.h. I mean if Windows has to get timeval from e_os.h, then it would be "prettier" if Unix did too...

Reply:
Good suggestion.  Unless someone objects, I'll look into doing this too.

Reply:
I'd suggest to do it in separate request. At least they moved socket stuff to separate file, so it might happen that it would be appropriate to move some place else. Myself I need to re-evaluate things with this new header, so I have no suggestions at this point.

Reply:
Okay, I'll leave this one for later.

Reply:
I created #4271 so this doesn't entirely disappear.